src/dapp/libexec/dapp/dapp---make-library-state

```diff
@@ -18,20 +18,11 @@ if [ "$DAPP_VERBOSE" ]; then set -x; fi
 # history tracking.)
 
 sender=0x00a329c0648769a73afac7f9381e08fb43dbea72
-nonce=1048576
+nonce=48576
```

Comment:
Do you know what caused this to change? The previous value was valid at some point.

Reply:
afaik, the value is not relevant, it just needs to match whats in dapp build. `dapp address` couldn't handle such a high nonce value though.